tests/test_message/event_sample.py

```diff
@@ -0,0 +1,204 @@
+    "pulls_url": "https://api.github.com/repos/brngylni/demo_repo/pulls{/number}",
+    "milestones_url": "https://api.github.com/repos/brngylni/demo_repo/milestones{/number}",
+    "notifications_url": "https://api.github.com/repos/brngylni/demo_repo/notifications{?since,all,participating}",
+    "labels_url": "https://api.github.com/repos/brngylni/demo_repo/labels{/name}",
+    "releases_url": "https://api.github.com/repos/brngylni/demo_repo/releases{/id}",
+    "deployments_url": "https://api.github.com/repos/brngylni/demo_repo/deployments",
+    "created_at": 1717013663,
+    "updated_at": "2024-07-22T21:14:26Z",
+    "pushed_at": 1721682985,
+    "git_url": "git://github.com/brngylni/demo_repo.git",
+    "ssh_url": "git@github.com:brngylni/demo_repo.git",
+    "clone_url": "https://github.com/brngylni/demo_repo.git",
+    "svn_url": "https://github.com/brngylni/demo_repo",
+    "homepage": None,
+    "size": 34,
+    "stargazers_count": 0,
+    "watchers_count": 0,
+    "language": None,
+    "has_issues": True,
+    "has_projects": True,
+    "has_downloads": True,
+    "has_wiki": False,
+    "has_pages": False,
+    "has_discussions": False,
+    "forks_count": 1,
+    "mirror_url": None,
+    "archived": False,
+    "disabled": False,
+    "open_issues_count": 0,
+    "license": None,
+    "allow_forking": True,
+    "is_template": False,
+    "web_commit_signoff_required": False,
+    "topics": [],
+    "visibility": "private",
+    "forks": 1,
+    "open_issues": 0,
+    "watchers": 0,
+    "default_branch": "main",
+    "stargazers": 0,
+    "master_branch": "main"
+  },
+  "pusher": {
+    "name": "brngylni",
+    "email": "66281850+brngylni@users.noreply.github.com"
+  },
+  "sender": {
+    "login": "brngylni",
+    "id": 66281850,
+    "node_id": "MDQ6VXNlcjY2MjgxODUw",
+    "avatar_url": "https://avatars.githubusercontent.com/u/66281850?v=4",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/brngylni",
+    "html_url": "https://github.com/brngylni",
+    "followers_url": "https://api.github.com/users/brngylni/followers",
+    "following_url": "https://api.github.com/users/brngylni/following{/other_user}",
+    "gists_url": "https://api.github.com/users/brngylni/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/brngylni/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/brngylni/subscriptions",
+    "organizations_url": "https://api.github.com/users/brngylni/orgs",
+    "repos_url": "https://api.github.com/users/brngylni/repos",
+    "events_url": "https://api.github.com/users/brngylni/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/brngylni/received_events",
+    "type": "User",
+    "site_admin": False
+  },
+  "created": False,
+  "deleted": False,
+  "forced": False,
+  "base_ref": None,
+  "compare": "https://github.com/brngylni/demo_repo/compare/034c31e8e464...443f68ce112a",
```

Comment:
Can this be emulated using VCR.py? 

Ref. https://vcrpy.readthedocs.io/

Reply:
I checked the documentation but it seems like there's a request, response structure when recording requests whereas we receive the request from GitHub to a webhook. So I'm not sure if we can make use of this. What do you think?

` import vcr
import urllib.request

with vcr.use_cassette('fixtures/vcr_cassettes/synopsis.yaml'):
    response = urllib.request.urlopen('http://www.iana.org/domains/reserved').read()
    assert b'Example domains' in response `

https://vcrpy.readthedocs.io/en/latest/usage.html#record-modes